qtmodern/windows.py

```diff
@@ -169,6 +168,39 @@ def setWindowTitle(self, title):
         super(ModernWindow, self).setWindowTitle(title)
         self.lblTitle.setText(title)
 
+    def _setWindowButtonState(self, button: QToolButton, state: bool):
```

Comment:
If I remember correcttly type hinting was introduced in 3.5? qtmodern tries to support earlier versions than this. So even though I like type hinting we can't use it in qtmodern

Reply:
removed them